cmd/gossamer/configcmd.go

```diff
@@ -46,15 +48,17 @@ import (
 
 // makeNode sets up node; opening badgerDB instance and returning the Dot container
 func makeNode(ctx *cli.Context) (*dot.Dot, *cfg.Config, error) {
-	fig, err := getConfig(ctx)
+	defaultConfig, err := getConfig(ctx)
```

Comment:
is this the defaultConfig? I don't think it's always the default

Reply:
cfg.DefaultConfig is the struct that is used to marshal the config into, so that is why I called it defaultConfig everywhere its used, its already better than fig IMHO.

Reply:
`cfg.DefaultConfig()` returns a copy of the config with all defaults applied, we then apply all specified options, so at this point it is not a default config

Reply:
ok, how about calling it currentConfig ?

Reply:
I agree, `currentConfig` would be probably be better or keeping it as it was.